scripts/load_mentors.js

```diff
@@ -99,11 +103,16 @@ exports.handler = async (event) => {
     const { id: mentorId } = result;
     if (!mentorIds.has(mentorId)) {
       oldMentorIds.push(mentorId);
```

Comment:
This is not related to your PR, but could you help to rename `oldMentorIds` to `deletedMentorIds` and also amend the log message at L114? Think this would make the code clearer.